frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh

```diff
@@ -1,37 +1,17 @@
-KUBEVIRT_VERSION=$(curl -s https://github.com/kubevirt/kubevirt/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
-#CDI_VERSION=$(curl -s https://github.com/kubevirt/containerized-data-importer/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
-CDI_VERSION="v1.29.0"
+#!/bin/bash
 
-VIRTCTL_DOWNLOAD_URL="https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/virtctl-${KUBEVIRT_VERSION}"
```

Comment:
if you remove the `VIRTCTL_DOWNLOAD_URL` how does it know where to get the virtctl command line from ?